src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -531,6 +546,66 @@ public List<User> getMatchedUsers(long userId) {
     return users;
   }
 
+
+  // instantiate demo users
+  public void instantiateDemoUsers() {
+
+    if (areInstantiatedDemoUsers) {
+      // throw exception if demo users are already instantiated
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Demo users are already instantiated");
+
+    } else {
+
+      System.out.println("Instantiating demo users...");
```

Comment:
I bet sonarlint would not be happy about this. Better use log.info as in cardService!